src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -187,6 +187,19 @@
     //  "ConsumerSecret": "",
     //  "AccessToken": "",
     //  "AccessTokenSecret": ""
+    //},
+    //"OrchardCore_MicrosoftAuthentication": {
+    //  "AppId": "",
+    //  "AppSecret": "",
+    //  "CallbackPath": "/",
```

Comment:
Please similarly populate `CallbackPath` in the other examples. Also, Use one that could be actually used (or would a login really work with just `/`?).

Reply:
This is just an example, the important thing is to start with `/`, can we use `/callback-url` for instance?

Reply:
Use the one that would work by default with each provider, so people don't need to change this part of the sample.

Reply:
Got it, but I'm not sure if there's a default callback path, because this depends on the app

Reply:
According to the [docs](https://docs.orchardcore.net/en/latest/docs/reference/modules/Twitter/), Twitter for instance uses `/signin-twitter` by default

Reply:
I will update the PR soon ..